src/arm/itx.h

```diff
@@ -117,9 +117,11 @@ static ALWAYS_INLINE void itx_dsp_init_arm(Dav1dInvTxfmDSPContext *const c, int
 
     if (!(flags & DAV1D_ARM_CPU_FLAG_NEON)) return;
 
+    assign_itx_fn( , 4, 4, wht_wht,           WHT_WHT,           neon);
+
     if (BITDEPTH == 16 && bpc != 10) return;
 
-    assign_itx17_fn( ,  4,  4, neon);
+    assign_itx16_fn( ,  4,  4, neon);
```

Comment:
Were we previously mistranslating this?  Because I don't see a previous `assignitx17_fn!` in our code.  Or had we broken the 17 out into a 16 and the `assign_itx_fn!(self, BD, 4, 4, wht_wht, WHT_WHT, neon);` call?

Reply:
`assign_itx17_fn` is a macro that invokes `assign_itx16_fn` and `assign_itx_fn`. It was already broken up in the Rust code.